README.md

```diff
@@ -262,18 +264,62 @@ func main() {
 }
 ```
 
-## Contribute
+### Send a mail with send API v 3.1
+
+```go
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	mailjet "github.com/mailjet/mailjet-apiv3-go"
+)
+
+func main() {
+	m := mailjet.NewMailjetClient(
+		os.Getenv("MJ_APIKEY_PUBLIC"),
+		os.Getenv("MJ_APIKEY_PRIVATE"))
+
+	messagesInfo := []mailjet.InfoMessagesV31{
+		mailjet.InfoMessagesV31{
+			From: &mailjet.RecipientV31{
```

Comment:
Semantically speaking, it just represent a Email/Name structure. A recipient has a special meaning in the email industry, meaning a person meant to receive an email message. Structure to rename